docs/openstack/mariadb_copy.md

```diff
@@ -111,4 +128,7 @@ SOURCE_MARIADB_IP=127.17.0.100
   ```
   oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
      mysql -h "$PODIFIED_MARIADB_IP" -uroot "-p$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;'
+  oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
+      mysql -h "$PODIFIED_CELL1_MARIADB_IP" -uroot "-p$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;' \
+      | grep nova
   ```
```

Comment:
do we also want to check for cell db mappings, connection/transport URIs etc. records? To compare it later on, if there were no templating issues introduced after adoption is done?

Reply:
i think that is out of scope

the intent of this PR is just to copy the data into the renamed db

updating the cell mapping is a separate step that will be likely implemented in the nova operator not in the adoption work.

i.e. when we deploy the conductors with the exting data i would expect the nova operator to correct the cell mappings with the new transport URL and db info so i don't think that should be covered in this pr.